### DISCUSSION TITLE
Height Integration

### DISCUSSION THREAD

**Discussion by edwarddarrah** [outline/outline#5391] [General]
Has there been a consideration in integrating [Height](https://height.app/) with Outline?

**Comment by tommoor** [comment]
What kind of integration are you hoping to see?

**Reply by edwarddarrah** [reply]
I would like to see more project management integrations. I know there are so many out there. I find it helpful to document with Outline and have related tasks embedded. I am currently using [Height](https://height.app/). 

Let me know if you have any specific questions. Thanks.
